src/components/RouteLink.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Fragment } from 'react';
+import { Link, useMatch, useResolvedPath } from 'react-router-dom';
+
+export const RouteLink = ({ children, to }) => {
+  let resolved = useResolvedPath(to);
+  // console.log('props---->', to);
```

Comment:
These are great for developer but I personally like to see code free of comments like this. They're easy to include when you need to test again and often become out of sync with the functioning code when refactored.